FilteringApp/FilteringApp.ConsoleUI/Program.cs

```diff
@@ -1,77 +1,87 @@
 ﻿using FilteringApp.Core.Models;
 using FilteringApp.Core.Utils;
-using CheckUserInput.Core;
 using System;
+using System.Collections.Generic;
 
 namespace FilteringApp.ConsoleUI
 {
     class Program
     {
         static void Main(string[] args)
         {
-            var arrayNumberOK = true;
-            var userValueOK = true;
-            var userInput = new UserInput();
-            //string[] stringArray = new String[0] { };
-            //int len = stringArray.Length;
-            //int[] array = new int[len];
-            int[] array2 = new int[0] { };
             Console.Clear();
-            do
-            {
-                Console.WriteLine("Please provide INT numbers for the array, separated by commas (,).");
-                string[] stringArray = Console.ReadLine().Split(',');
-                int len = stringArray.Length;
-                int[] array = new int[len];
 
-                for (int i = 0; i < len; i++)
-                {
-                    //array[i] = Convert.ToInt32(stringArray[i]);
-                    string v = stringArray[i];
-                    var ov = 0;
-                    if (!int.TryParse(v, out ov))
-                    {
-                        Console.WriteLine("Wrong Input in array: " + v + " Postion :" + (i + 1));
-                        arrayNumberOK = false;
-                    }
-                    else
-                    {
-                        array[i] = Convert.ToInt32(stringArray[i]);
-                        arrayNumberOK = true;
-                    }
+            int[] array = new int[0];
+            //int intValue;
+
+            
```

Comment:
Please remove all the multi-line empty spaces

Reply:
Fixed.